docs/migration_guide.md

```diff
@@ -1,17 +1,20 @@
 # Harbor upgrade and database migration guide
 
-When upgrading your existing Habor instance to a newer version, you may need to migrate the data in your database and the settings in harbor.cfg. Refer to [change log](../tools/migration/changelog.md) to find out whether there is any change in the database. If there is, you should go through the database migration process. Since the migration may alter the database schema and the settings of harbor.cfg, you should **always** back up your data before any migration.
+When upgrading your existing Harbor instance to a newer version, you may need to migrate the data in your database and the settings in `harbor.cfg`. Refer to [change log](../tools/migration/db/changelog.md) to find out whether there is any change in the database. If there is, you should go through the database migration process. Since the migration may alter the database schema and the settings of `harbor.cfg`, you should **always** back up your data before any migration.
 
-*If your install Harbor for the first time, or the database version is the same as that of the lastest version, you do not need any database migration.*
+*If your install Harbor for the first time, or the database version is the same as that of the latest version, you do not need any database migration.*
 
 **NOTE:**
-- From v1.6.0 on, Harbor migrates DB from MariaDB to Postgresql, and combines Harbor, Notary and Clair DB into one. 
+- This guide only covers the migration from v1.6.0 to current version, if you are upgrading from earlier versions please 
+refer to the migration guide in release branch to upgrade to v1.6.0 and follow this guide to do the migration to later version. 
```

Comment:
nit: erroneous `\n`

Reply:
As for the `\n\` I 've made update so that there is a `\n` between each item.